core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -98,6 +99,8 @@ type plugin struct {
 	TokenDataEncoder    cciptypes.TokenDataEncoder
 	GasEstimateProvider cciptypes.EstimateProvider
 	RMNCrypto           func(lggr logger.Logger) cciptypes.RMNCrypto
+	// PriceOnlyCommitFn optional method override for price only commit reports.
+	PriceOnlyCommitFn string
```

Comment:
Where/how would this get set?

Reply:
It's set a few lines above. Hard coded for the Solana chain family.